src/Avalonia.Controls/Platform/DefaultMenuInteractionHandler.cs

```diff
@@ -275,7 +275,7 @@ protected internal virtual void AccessKeyPressed(object sender, RoutedEventArgs
                 return;
             }
 
-            if (item.HasSubMenu)
+            if (item.HasSubMenu && item.IsEnabled)
```

Comment:
Not sure, but IsEffectivelyEnabled seems to be correct here

Reply:
Makes sense